contrib/babelfishpg_tsql/src/session.c

```diff
@@ -211,17 +211,17 @@ Datum babelfish_db_name(PG_FUNCTION_ARGS)
 	if (dbid == 1)
 	{
 		dbname = palloc((strlen("master") + 1) * sizeof(char));
-		strncpy(dbname, "master", sizeof("master"));
+		strncpy(dbname, "master", strlen("master") + 1);
```

Comment:
sizeof("master") should be same as strlen("master") + 1. So why do we need this change?